stemcell_builder/stages/bosh_users/apply.sh

```diff
@@ -18,8 +18,8 @@ run_in_chroot $chroot "
 groupadd --system admin
 useradd -m --comment 'BOSH System User' vcap
 chmod 755 ~vcap
-echo \"vcap:${bosh_users_password}\" | chpasswd
-echo \"root:${bosh_users_password}\" | chpasswd
+echo "vcap:${bosh_users_password}" | chpasswd
```

Comment:
With escape character it is not able to change password in Photon. I verified in Fedora and Ubuntu with out escape character and it works. Let me know if I have to revert this change.